README.md

```diff
@@ -1,31 +1,62 @@
 [RethinkDB](http://www.rethinkdb.com)
 =================
 
-RethinkDB is an open-source distributed database. It has an intuitive
```

Comment:
I think this paragraph needs the most work because it's using the old story. Here is an example you might want to use instead:

> RethinkDB is the first open-source scalable database for the realtime web. It exposes a new database access model -- instead of polling for changes, the developer can tell the database to continuously push updated query results to applications in realtime. RethinkDB allows developers to build scalable realtime apps in a fraction of the time with less effort.